projects/ion/src/lib/picker/date-picker/date-picker-calendar/date-picker-calendar.component.html

```diff
@@ -5,16 +5,25 @@
     </span>
   </div>
   <div class="month-days">
-    <button
-      [attr.data-testid]="getAriaLabel(day)"
+    <div
       *ngFor="let day of days; let i = index"
-      class="month-day"
-      [class.current]="day.isDayCurrentMonth"
-      [class.selected]="isSelectedDate(day)"
-      [attr.aria-label]="getAriaLabel(day)"
-      (click)="dispatchActions(i)"
+      [attr.data-testid]="'container-' + getAriaLabel(day)"
+      [class.between-range]="day.isBetweenRange"
```

Comment:
Quando o calendário é configurando como `rangePicker` e a data início e fim são iguais, a barra do range aparece. 
![image](https://github.com/Brisanet/ion/assets/138057453/e04b5be9-6a0a-4c00-8611-dcaa7288dd54)
Acredito que adicionando uma validação ao configurar a classe `between-range`, deve resolver.